addons/kubeadm-configmap/kubeadm-configmap.yaml

```diff
@@ -23,12 +23,20 @@ data:
       extraArgs:
         authorization-mode: Webhook
       timeoutForControlPlane: 4m0s
-    apiVersion: kubeadm.k8s.io/v1beta1
+    {{- if and (eq .Cluster.Version.Major 1) (le .Cluster.Version.Minor 21) }}
```

Comment:
Wouldn't `.Cluster.Version.LessThan "1.22"` also work?

Reply:
Also, why do we need an `else if` and not just an `else`?

Reply:
I haven't worked with the semver library too much, but my understanding is that `LessThan` needs another `Version` (https://pkg.go.dev/github.com/Masterminds/semver/v3#Version.LessThan), so I don't think passing a string (`"1.22"`) would work? Or is that incorrect?

The else is a good point, let me simplify it.